### PR TITLE
Port over hopsuit

### PR DESCRIPTION
**Pull request by Finket**
# Description

Port over https://github.com/Simple-Station/Parkstation/pull/87
With changes from https://github.com/Simple-Station/Parkstation/pull/105

- [x] HoP Hardsuit https://github.com/Simple-Station/Parkstation-Friendly-Chainsaw/issues/2

Note there are other changes in the original Hopsuit PR that I'll just list as tasks here just in case. I did not make those changes.

---

# Tasks

- [ ] command encryption key box
- [ ] command key box in HoP locker
- [ ] command remote in HoP locker

---

<details><summary><h1>Media</h1></summary>
<p>

image

image

</p>
</details>

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml [hunk 1]
```diff
@@ -135,6 +135,7 @@
       - id: ClothingUniformJumpsuitBoatswain # DeltaV - Add turtleneck for HoP.
       - id: ClothingShoesBootsLaceup # DeltaV - add fancy shoes for HoP and cap
       - id: ClothingShoesMiscWhite # DeltaV - add fancy shoes for HoP and cap
+      - id: ClothingOuterHardsuitHoP
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:138]:
```suggestion
      - id: ClothingOuterHardsuitHoP # Parkstation-HoPSuit
```

File: Resources/Textures/SimpleStation14/Clothing/Head/Hardsuits/hopsuit.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Made by Cabbage#9633",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Clothing/Head/Hardsuits/hopsuit.rsi/meta.json:4]:
```suggestion
    "copyright": "Made by DSC@Cabbage#9633",
```

File: Resources/Textures/SimpleStation14/Clothing/OuterClothing/Hardsuits/hopsuit.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+﻿{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Made by Cabbage#9633",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Clothing/OuterClothing/Hardsuits/hopsuit.rsi/meta.json:4]:
```suggestion
    "copyright": "Made by DSC@Cabbage#9633",
```

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

**Comment by Finket** [comment]
Requested changes committed.

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Pull request merged**